Analysis/Utkscan/processors/include/PspmtProcessor.hpp

```diff
@@ -1,38 +1,122 @@
-/** \file PspmtProcessor.hpp
- *  \brief A processor to handle pixelated PMTs
- *  \author Shintaro Go
- *  \date November 16, 2015
- */
-
+///@file PspmtProcessor.cpp
+///@brief Processes information from a Position Sensitive PMT.
+///@authors S. Go and S. V. Paulauskas
+///@date August 24, 2016
 #ifndef __PSPMTPROCESSOR_HPP__
 #define __PSPMTPROCESSOR_HPP__
 
-#include "RawEvent.hpp"
+#include <string>
+#include <map>
+
 #include "EventProcessor.hpp"
+#include "RawEvent.hpp"
 
 ///Class to handle processing of position sensitive pmts
 class PspmtProcessor : public EventProcessor {
 public:
-    /** Default Constructor */
-    PspmtProcessor(void);
-    /** Default Destructor */
+    ///Default Constructor */
+    PspmtProcessor(void) {};
+
+    ///@brief Constructor that sets the scale and offset for histograms
+    ///@param[in] scale : The multiplicative scaling factor
+    ///@param[in] offset : The additave offset for the histogram
+    PspmtProcessor(const std::string &vd, const double &scale,
```

Comment:
this is my bad i had him correct to 80. i saw that you had discussed it but not that it was implemented